--- cmake/modules/DispatchCompilerWarnings.cmake.orig	2022-03-23 15:34:30.474065670 -0500
+++ cmake/modules/DispatchCompilerWarnings.cmake	2022-03-23 15:46:39.228684206 -0500
@@ -2,7 +2,6 @@
 if("${CMAKE_C_SIMULATE_ID}" STREQUAL "MSVC")
   # TODO: someone needs to provide the msvc equivalent warning flags
 else()
-  add_compile_options($<$<OR:$<COMPILE_LANGUAGE:C>,$<COMPILE_LANGUAGE:CXX>>:-Werror>)
   add_compile_options($<$<OR:$<COMPILE_LANGUAGE:C>,$<COMPILE_LANGUAGE:CXX>>:-Wall>)
   add_compile_options($<$<OR:$<COMPILE_LANGUAGE:C>,$<COMPILE_LANGUAGE:CXX>>:-Wextra>)
 
